### DISCUSSION TITLE
A couple of UI questions

### DISCUSSION THREAD

**Discussion by tunnus-1** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#703] [General]
When using a desktop browser, calibration hint is only partially shown. See below:

![Screenshot 2024-05-11 at 18 52 09](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/74300867/80d086cb-b010-4b28-ab13-789a8c88bd93)

Also, I'm using (or at least I think I'm using :) release v4.2.5, dated 1st of March, but UI is showing a date from 2023. Again, see below:

![Screenshot 2024-05-11 at 18 52 37](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/74300867/6f57d7c3-51a5-48d2-ae62-b98570302bc6)

Would it be possible to add a version number to UI?

**Comment by tunnus-1** [comment]
@visualapproach - Not overly concerned about that text clipping, but what about version number, any chance it could be added?

**Reply by visualapproach** [reply]
Not easily. Firmware date-time is set in the code. Version is just a GitHub label that I attach when moving dev branch into master. There is probably a workflow for that but I settled with this way after some initial headaches with mismatching version numbering

**Comment by metaljay** [comment]
@colinpeddle can you take a look possibly?

**Comment by visualapproach** [comment]
tooltip is moved now in dev branch :-)
